angularSTEP/src/app/profile-card/profile-card.component.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { async, ComponentFixture, TestBed, inject, tick, fakeAsync } from '@angular/core/testing';
+import { DebugElement, Component } from '@angular/core';
+
+import { By } from '@angular/platform-browser';
```

Comment:
I don't see this used anywhere; does it really need to be imported?

Reply:
I was planning too, but couldn't find a good use for it with this component